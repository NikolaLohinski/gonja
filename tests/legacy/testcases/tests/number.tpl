{{ 42 is number }}
{{ 42.0 is number }}
{{ 'gonja' is number }}
{{ number is number }}
{{ simple.number is number }}
{{ simple.uint is number }}
{{ simple.float is number }}
{{ simple.nil is number }}
{{ simple.str is number }}
{{ None is number }}
