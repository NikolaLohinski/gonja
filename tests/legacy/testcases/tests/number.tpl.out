True
True
False
True
True
True
True
False
False
False
